test/evergreen.yml

```diff
@@ -5988,7 +6003,10 @@ buildvariants:
     - name: format-stress-pull-request-test
     - name: make-check-test
       distros: ubuntu2004-large
-    - name: csuite-long-running
+    - name: csuite-long-running-subset1
+      distros: ubuntu2004-large
```

Comment:
is there a reason behind using a large ubuntu distro here?

Reply:
That was something I left unchanged from the previous version of the test.

Reply:
It's worth seeing if those tests can run ok on a smaller distro, so I'll try that in a patch build.

Reply:
I ran a patch build with a default (smaller) instance type, and one of the tests failed.
I've added a comment in each task explaining that a 'large' distro is required to run all the tests successfully.